assets/js/langpack/es.json

```diff
@@ -0,0 +1,126 @@
+{
+  "token": {
+    "Claim control of your":
+    "Obtén el control de tus",
+
+    "Docker images":
+    "Imágenes Docker",
+
+    "View on GitHub":
+    "Ver en GitHub",
+
+    "Download .zip":
+    "Descargar .zip",
+
+    "Download .tar.gz":
+    "Descargar .tar.gz",
+
+    "What is Portus":
+    "Qué es Portus",
+
+    "Portus is an open source authorization service and user interface for the next generation Docker Registry.":
+    "Portus es un servicio de autorización de libre código e interfaz de usuario para la próxima generación de Registros Docker",
```

Comment:
Remove "libre código" since:

1. It's "código libre" in spanish
2. We don't say that in the english version.